polaris-for-vscode/scripts/getGroupedCustomPropertyCompletionItems.js

```diff
@@ -61,7 +42,4 @@ const getGroupedCustomPropertyCompletionItems = () => {
   );
 };
 
-module.exports = {
-  getGroupedCustomPropertyCompletionItems,
-  getCustomPropertyNames,
-};
+module.exports = getGroupedCustomPropertyCompletionItems;
```

Comment:
I recommend leaving this as `module.exports = { getGroupedCustomPropertyCompletionItems }`. I've been having a lot of headaches lately with es module interop and "named exporting" all the things seem to be the silver bullet.